features/commit.feature

```diff
@@ -15,3 +17,10 @@ Feature: Creating a git commit from a Pivotal Tracker story
       # No optional message
       And I type ""
     Then the output should contain "> No staged changes. Will create an empty commit."
+
+  Scenario: With an empty global config file, the user is prompted for their PT API key
+    Given a file named "tmp/global_settings.yml" with "--- {}\n"
+    When I run `geordi commit` interactively
+      And I type "abc"
+      And I type ""
+    Then the file "tmp/global_settings.yml" should contain "pivotal_tracker_api_key: abc"
```

Comment:
Please do not clear the settings file, but remove it instead. This test covers the current situation of all Geordi users lacking the global settings file. The scenario should be titled "Without a global config file ...".

A test should explain itself as best as possible. Please replace "abc" with "my tracker api key" and fill in the empty quotes with "optional message".